plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -128,11 +140,17 @@ public class TrinoGlueCatalog
     private static final Logger LOG = Logger.get(TrinoGlueCatalog.class);
 
     private final String trinoVersion;
+    private final TypeManager typeManager;
+    private final boolean cacheTableMetadata;
     private final TrinoFileSystemFactory fileSystemFactory;
     private final Optional<String> defaultSchemaLocation;
     private final AWSGlueAsync glueClient;
     private final GlueMetastoreStats stats;
 
+    private final Cache<SchemaTableName, com.amazonaws.services.glue.model.Table> glueTableCache = EvictableCacheBuilder.newBuilder()
+            // Even though this is query-scoped, this still needs to be bounded. information_schema queries can access large number of tables.
+            .maximumSize(Math.max(1000, IcebergMetadata.GET_METADATA_BATCH_SIZE))
```

Comment:
Math.max is unnecessary, always returns a constant value

Reply:
I know. It was supposed to convey the following logic "size the cache picking some 1000 as default, but not less than GET_METADATA_BATCH_SIZE, otherwise bad things will happen"